src/serverMiddleware/routes.js

```diff
@@ -7,8 +7,8 @@
 
 module.exports = [
 	`^/administration/datasources?`,
-	// `^/administration/students/?$`,
-	// `^/administration/teachers/?$`,
+	`^/administration/students/?$`,
+	`^/administration/teachers/?$`,
```

Comment:
Das bitte erstmal nicht commiten. Aktivieren wir dann später erst, wenn die Seiten fertig sind.
```suggestion
	// `^/administration/students/?$`,
	// `^/administration/teachers/?$`,
```

Reply:
alles klar